usaspending_api/common/serializers.py

```diff
@@ -78,6 +77,38 @@ def __init__(self, *args, **kwargs):
                     # We don't have get default fields available
                     pass
 
+    '''
```

Comment:
Moving the comment to what is now known as line 93 (re: after `def_setup_eager_loading`) will make it a [docstring](https://www.python.org/dev/peps/pep-0257/)